packages/terra-i18n/docs/README.md

```diff
@@ -28,6 +53,24 @@ Note that the state of the object needs to contain keys as follows for the i18nL
 }
 ```
 
+Additionally, do not render translated components until areTranslationsLoaded is true.
```

Comment:
Nit: I feel its a little confusing. Could we move this explanation to be an inline comment of the provided example?

Reply:
Updated and rebased. Good recommendation.